erpnext/assets/doctype/asset_repair_consumed_item/asset_repair_consumed_item.json

```diff
@@ -44,12 +30,26 @@
    "in_list_view": 1,
    "label": "Item",
    "options": "Item"
+  },
+  {
+   "fetch_from": "item.valuation_rate",
+   "fieldname": "rate",
```

Comment:
These new fields won't get auto patched. You'll have to write patch to migrate existing data.